api/src/main/java/io/grpc/ServerBuilder.java

```diff
@@ -89,6 +90,16 @@
    */
   public abstract T addService(BindableService bindableService);
 
+  /**
+   * Adds a list of service implementations to the handler registry together.
+   *
+   * @param bindableServices the list of BindableService objects
+   * @return this
+   * @since 1.37.0
+   */
+  @ExperimentalApi("https://github.com/grpc/grpc-java/issues/7925")
+  public abstract T addServices(List<BindableService> bindableServices);
```

Comment:
I think we'd need to add a default implementation here to preserve API stability. It doesn't have to work (it could throw), but it seems we might as well just implement it.